modules/notifications/notifications.php

```diff
@@ -4,1292 +4,1509 @@
+		/**
+		 * Add the subscriptions meta box to relevant post types
+		 */
+		public function add_post_meta_box() {
+
+			if ( ! current_user_can( $this->edit_post_subscriptions_cap ) ) {
+				return;
+			}
 
-		return array(
-				'title'     => $title_text,
-				'text'      => $follow_text,
-				'link'      => add_query_arg( $args, admin_url( 'admin-ajax.php' ) ),
-			);
-	}
-	
-	/**
-	 * Add the subscriptions meta box to relevant post types
-	 */
-	function add_post_meta_box() {
-
-		if ( !current_user_can( $this->edit_post_subscriptions_cap ) ) 
-			return;		
-		
-		$usergroup_post_types = $this->get_post_types_for_module( $this->module );
-		foreach ( $usergroup_post_types as $post_type ) {
-			add_meta_box( 'edit-flow-notifications', __( 'Notifications', 'edit-flow'), array( $this, 'notifications_meta_box'), $post_type, 'advanced' );
+			$usergroup_post_types = $this->get_post_types_for_module( $this->module );
+			foreach ( $usergroup_post_types as $post_type ) {
+				add_meta_box( 'edit-flow-notifications', __( 'Notifications', 'edit-flow' ), [ $this, 'notifications_meta_box' ], $post_type, 'advanced' );
+			}
 		}
-	}
-	
-	/**
-	 * Outputs box used to subscribe users and usergroups to Posts
-	 *
-	 * @todo add_cap to set subscribers for posts; default to Admin and editors
-	 */	
-	function notifications_meta_box() {
-		global $post, $post_ID, $edit_flow;
-		?>
-		<div id="ef-post_following_box">
-			<a name="subscriptions"></a>
-
-			<p><?php _e( 'Select the users and user groups that should receive notifications when the status of this post is updated or when an editorial comment is added.', 'edit-flow' ); ?></p>
-			<div id="ef-post_following_users_box">
-				<h4><?php _e( 'Users', 'edit-flow' ); ?></h4>
-				<?php
-				$followers = $this->get_following_users( $post->ID, 'id' );
-				$select_form_args = array(
-					'list_class' => 'ef-post_following_list',
-				);
-				$this->users_select_form( $followers, $select_form_args ); ?>
-			</div>
-			
-			<?php if ( $this->module_enabled( 'user_groups' ) && in_array( $this->get_current_post_type(), $this->get_post_types_for_module( $edit_flow->user_groups->module ) ) ): ?>
-			<div id="ef-post_following_usergroups_box">
-				<h4><?php _e('User Groups', 'edit-flow') ?></h4>
-				<?php
-				$following_usergroups = $this->get_following_usergroups( $post->ID, 'ids' );
-				$edit_flow->user_groups->usergroups_select_form( $following_usergroups ); ?>
+
+		/**
+		 * Outputs box used to subscribe users and usergroups to Posts
+		 *
+		 * @todo add_cap to set subscribers for posts; default to Admin and editors
+		 */
+		public function notifications_meta_box() {
+			global $post, $post_ID, $edit_flow;
+			?>
+			<div id="ef-post_following_box">
+				<a name="subscriptions"></a>
+
+				<p><?php _e( 'Select the users and user groups that should receive notifications when the status of this post is updated or when an editorial comment is added.', 'edit-flow' ); ?></p>
```

Comment:
We should add email notifications here so it's clear that webhook doesn't count.

Reply:
Changed in 902bc0473ba0f728ba2b6f95abbf208be1e71650